src/NumberGuessingGame.js

```diff
@@ -15,7 +15,6 @@ function getRandomNumber() {
 
 const MAX_ATTEMPTS = 5;
 
-    //convert NumberGuessingGame class to a function
   const NumberGuessingGame = (props) => {
```

Comment:
nit: extra indentation. And if you wanted you could destructure your props so you don't have to reference props.onGuess

Reply:
thx